src/lib/Parser.ts

```diff
@@ -0,0 +1,59 @@
+import { ParserOptions, ParsedAst, FormatterOptions } from './types'
+import { Parser } from 'node-sql-parser/build/postgresql'
+import prettier from 'prettier/standalone'
+import SqlFormatter from 'prettier-plugin-sql'
+
+const DEFAULT_PARSER_OPTIONS = {
+  database: 'PostgreSQL',
+  type: 'table',
+}
+
+const DEFAULT_FORMATTER_OPTIONS = {
+  plugins: [SqlFormatter],
+  formatter: 'sql-formatter',
+  language: 'postgresql',
+  database: 'postgresql',
+  uppercase: false, // I thought this would "lowercase" everything, but it doesn't. There is not option to convert everything to lowercase.
```

Comment:
I would love an option to lowercase the values. it would require a fork of this: https://github.com/zeroturnaround/sql-formatter (which hasn't been active since Feb 2021)